examples/direct-upload/README.md

```diff
@@ -0,0 +1,59 @@
+## Direct Upload Example
+
+We want to upload images to an S3 bucket via an html form.
+
+html form -
+
+The html form needs to have a meta tag in the head that takes the following to
+let the browser know that you're using UTF-8 unicode character encoding:
+
+```html
+<meta http-equiv="content-type" content="text/html" charset="utf-8" />
+```
+
+The form needs to send certain information to S3 before the upload can be accepted.
+If any information is excluded or incorrect the upload will be rejected.
+
++ #### Step 1:
+Assign the action of your form to the bucket URL you want to upload to.
+If you're new to Amazon Web Services, you'll first have to create an account on the
+[Amazon Developer Console](aws.amazon.co.uk) Navigate to the S3 console and create
+your first bucket. To access the bucket URL simply use this format with your
+bucket name `https://[your-bucket-name].s3.amazonaws.com/`. Alternatively you can
+include the region explicitly but AWS http://bucket.s3-aws-region.amazonaws.com
+will be able to find it from the bucket name automatically.
+
+```html
+<body>
+  <form action="https://[your-bucket-name].s3.amazonaws.com/" method="post" enctype="multipart/form-data">
+```
+
++ #### Step 2:
```

Comment:
bullet-pointing sub-headings negates the effect of the sub-heading ... just looks like you're `hashtag-ing` it... 😜